vendor.conf

```diff
@@ -1,7 +1,7 @@
 # the following lines are in sorted order, FYI
 github.com/Azure/go-ansiterm 388960b655244e76e24c75f48631564eaefade62
 github.com/Microsoft/hcsshim v0.5.12
-github.com/Microsoft/go-winio v0.3.8
+github.com/Microsoft/go-winio 7c7d6b461cb10872c1138a0d7f3acf9a41b5c353 https://github.com/dgageot/go-winio.git
```

Comment:
Can you add a `TODO` here, and a link to the PR, so that we know when we can remove this temporary fix?